api/can_api_definition.py

```diff
@@ -188,10 +188,33 @@ def check_timeseries_have_cumulative_test_data(cls, rows, values):
 
         return rows
 
+    @pydantic.validator('timeseries')
+    def check_timeseries_one_row_per_date(cls, rows, values):
+        dates_in_row = len(set(row.date for row in rows))
+        if len(rows) != dates_in_row:
+
+            raise ValueError(
+                "Number of rows does not match number of dates: "
+                f"{len(rows)} vs. {dates_in_row}"
+            )
+
+        return rows
+
 
 class CovidActNowCountyTimeseries(CovidActNowCountySummary):
     timeseries: List[CANPredictionTimeseriesRow] = pydantic.Field(...)
 
+    @pydantic.validator('timeseries')
+    def check_timeseries_one_row_per_date(cls, rows, values):
```

Comment:
does this have to be duplicated?

Reply:
I think since the county and state are separate pydantic models we need to.  can probably define the logic in some similar spot, but not sure there's a single point that works super easy